content/4_Gold/Modular.mdx

```diff
@@ -81,6 +81,13 @@ multiply as normal.
 	>
 		some practice probs
 	</Resource>
+    <Resource
+        source="MONT@Modern Olympiad Number Theory"
+        title="2 and 5 - Basic to Advanced Modular Arithmetic"
```

Comment:
`2 and 5` -> `2, 5` (like the other sources)

Reply:
done

Reply:
What about now?